openrndr-core/src/test/kotlin/TestDrawContour.kt

```diff
@@ -1,10 +1,9 @@
+import io.kotest.core.spec.style.DescribeSpec
 import org.openrndr.application
 import org.openrndr.shape.Circle
 import org.openrndr.shape.LineSegment
-import org.spekframework.spek2.Spek
-import org.spekframework.spek2.style.specification.describe
 
-object TestVertexBufferGL3 : Spek({
+class TestVertexBufferGL3 : DescribeSpec({
```

Comment:
Oh, just seeing, this change from object to class wasn't intended. Doesn't seem to make a difference though.

```suggestion
object TestVertexBufferGL3 : DescribeSpec({
```